src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -752,7 +752,11 @@ private boolean closeTabs() {
     }
 
     public boolean closeCurrentTab() {
-        return closeTab(getCurrentLibraryTab());
+        LibraryTab currentTab = getCurrentLibraryTab();
+        if (currentTab == null) {
+            return false;
+        }
+        return closeTab(currentTab);
```

Comment:
I would suggest moving this check on null to CloseDatabaseAction#execute, then this will be consistent with other actions and we keep the view (yes i know, right now according to the mvvm pattern, jabrefframe is still way off a pure view, but it gets there eventually).

Reply:
That makes sense. But in this case, do you suggest to call the `getCurrentLibraryTab` on `tabContainer` before calling `closeCurrentTab` ? That would make two calls to `getCurrentLibraryTab`.
Or maybe I should implement a method `getTabCount` to know if there is at least one tab.

Reply:
No, just return from the execute method (fail fast) with a log message if the the library tab and the currently opened tab is null.